templates/azure/terraform/flatten_property_method.erb

```diff
@@ -21,7 +21,36 @@
                            prefix: sdk_marshal.resource,
                            property: property)) -%>
 <% else -%>
-<% if tf_types.include?(property.class) -%>
+<% if property.is_a?(Api::Type::Array) && property.item_type == 'Api::Type::Integer'
+     bit_number = 32 if sdk_marshal.sdktype.type_definition.is_a?(Api::Azure::SDKTypeDefinition::Integer32ArrayObject)
+     bit_number = 64 if sdk_marshal.sdktype.type_definition.is_a?(Api::Azure::SDKTypeDefinition::Integer64ArrayObject)
+-%>
+func flattenIntegerArray(input *[]int<%= bit_number -%>) []interface{} {
```

Comment:
Similar to the comment above, we can abstract this into a common function.

Reply:
Done.